### DISCUSSION TITLE
Use Django Ninja

### DISCUSSION THREAD

**Discussion by joaoflaviosantos** [deta/cloud-docs#340] [General]
Can I install Django to use with DjangoNinja to serve an API?

**Comment by joaoflaviosantos** [comment]
Deploying...
Successfully deployed changes
Updating dependencies...


Error: failed to update dependecies: error on one or more dependencies, no dependencies were added, see output for details

**Reply by BrahianVT** [reply]
same error fixed with "https://docs.deta.sh/docs/common_issues/"

**Reply by Sokhavuth** [reply]
I think we could use Django on Deta platform by using SQLite database for read only. By default, Django comes with SQLite as its default database. And Deta platform supports SQLite for read only. I will try it someday.

**Comment by malwaremanu** [comment]
for Django you can create a file "main.py" and 
put below mentioned content  

```
from django_project.wsgi import application as app
```
